src/main/java/com/minecolonies/coremod/commands/colonycommands/MakeNotAutoDeletable.java

```diff
@@ -0,0 +1,99 @@
+package com.minecolonies.coremod.commands.colonycommands;
+
+import com.minecolonies.api.colony.permissions.Rank;
+import com.minecolonies.coremod.colony.Colony;
+import com.minecolonies.coremod.colony.ColonyManager;
+import com.minecolonies.coremod.commands.AbstractSingleCommand;
+import net.minecraft.command.CommandException;
+import net.minecraft.command.ICommandSender;
+import net.minecraft.entity.player.EntityPlayer;
+import net.minecraft.server.MinecraftServer;
+import net.minecraft.util.math.BlockPos;
+import net.minecraft.util.text.TextComponentString;
+import org.jetbrains.annotations.NotNull;
+
+import javax.annotation.Nullable;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
+
+public class MakeNotAutoDeletable extends AbstractSingleCommand
+{
+    public static final  String DESC                       = "undeletable";
+    private static final String NO_COLONY_FOUND_MESSAGE_ID = "Colony with ID %d not found.";
+    private static final String MARKED                     = "Marking succesful!";
+    private static final String NOT_ENOUGH_ARGUMENTS       = "You must have 2 Arguments: <ColonyId> <true|false> ";
+
+    private static final int NUMBER_OR_ARGS_REQUIRED = 2;
+
+    /**
+     * Initialize this SubCommand with it's parents.
+     *
+     * @param parents an array of all the parents.
+     */
+    public MakeNotAutoDeletable(@NotNull final String... parents)
+    {
+        super(parents);
+    }
+
+    @NotNull
+    @Override
+    public String getCommandUsage(@NotNull final ICommandSender sender)
+    {
+        return super.getCommandUsage(sender) + "<ColonyId> <true|false>";
+    }
+
+    @Override
+    public boolean canRankUseCommand(@NotNull final Colony colony, @NotNull final EntityPlayer player)
+    {
+        return colony.getPermissions().getRank(player).equals(Rank.OWNER);
+    }
+
+    @Override
+    public void execute(@NotNull final MinecraftServer server, @NotNull final ICommandSender sender, @NotNull final String... args) throws CommandException
+    {
+        if (args.length < NUMBER_OR_ARGS_REQUIRED)
+        {
+            sender.addChatMessage(new TextComponentString(NOT_ENOUGH_ARGUMENTS));
+            return;
+        }
+
+        sender.addChatMessage(new TextComponentString(Arrays.toString(args)));
+
+        int colonyId;
+        colonyId = Integer.parseInt(args[0]);
+        final Colony colony = ColonyManager.getColony(colonyId);
```

Comment:
i can't see any "isAdmin" or anything of the sort (I checked all the commands)
there is:
@Override
    public boolean canRankUseCommand(@NotNull final Colony colony, @NotNull final EntityPlayer player)
    {
        return colony.getPermissions().getRank(player).equals(Rank.OWNER);
    }

on commands like /mc colony delete , which is what i put on mine. but that's all i can find =/